### PR TITLE
Somethings afoot er

### PR DESCRIPTION
**Pull request by ericronne**
Initial footer reformatting. Do Not Merge

@jseppi

### CODE REVIEW

**Comment by jseppi** [comment]
hrm, it looks like maybe this PR was branched off of some old code or something (notice all the hbillings commits listed in the PR history), which is leading to some merge conflicts.
I'll see what I can do about cleaning that up.

**Comment by ericronne** [comment]
That's my biggest frustration with the github app :(
sry

**Comment by jseppi** [comment]
When logged-in, should the introductory text and header be removed?

screen shot 2016-12-27 at 3 46 53 pm

**Comment by jseppi** [comment]
I think I got `develop` properly merged into this one

**Comment by jseppi** [comment]
Also, should the logged-in header links be removed in tandem with these footer changes?

screen shot 2016-12-27 at 3 59 58 pm

**Comment by jseppi** [comment]
We might also want to have a strategy to deal with long email addresses. As currently implemented, a long email address will overflow the left "column" of the footer:

screen shot 2016-12-27 at 4 12 31 pm

**Comment by ericronne** [comment]
Good points @jseppi, thx! I killed the intro text for logged-in users. Also i'm inclined to remove the "Hello …" links from the top, and test it ( @tram :) ) . My theory is that users won't have trouble finding the logout/account links at the bottom, since that's where they logged in. Bigger question is whether they'll find them *before* they log in!

Solving spillover for long emails is beyond my current css ken, so i'll make another issue for that.

**Comment by jseppi** [comment]
I think this looks good.

We can handle removing the header links in a separate PR.

File: data_explorer/templates/_footer.html [hunk 1]
```diff
@@ -3,37 +3,32 @@
...
+            {% if not user.is_authenticated %}
+            <h2>Log in to add price lists</h2>
+            <p>Now contracting officers and other acquisition professionals can upload price lists to CALC.</p>
+              <div class="nav_wrap_footer">
+                <div class="button_footer">
+                  <p><a href="{% url 'uaa_client:login' %}">Log in</a></p>
+                </div>
+              </div>
+            {% elif user.is_authenticated %}
+                  <p>You are logged in as {{ user.email }}</p>
```
Comment (jseppi) [inline data_explorer/templates/_footer.html]: de-indent by one indentation

File: data_explorer/templates/_footer.html [hunk 2]
```diff
@@ -3,37 +3,32 @@
...
-                <!-- <li><a href="/docs/">Read the API documentation</a></li> -->
-              </ul>
-              {% if request.path == "/" %}
-                <h6>Icon credits</h6>
-                <p class="small">Paulo Sá Ferreira, Luis Prado, John Testa, Andrew Forrester from <a href="https://thenounproject.com/">The Noun Project</a>.</p>
-              {% endif %}
-            </div><!-- /.footer_nav -->
+          <div class="four columns">
+            {% if not user.is_authenticated %}
+            <h2>Log in to add price lists</h2>
```
Comment (jseppi) [inline data_explorer/templates/_footer.html]: indent by one indentation

**Comment by jseppi** [comment]
Actually made a few minor review comments to be addressed.

File: data_explorer/templates/_footer.html [hunk 3]
```diff
@@ -3,37 +3,32 @@
...
+            <p>Now contracting officers and other acquisition professionals can upload price lists to CALC.</p>
+              <div class="nav_wrap_footer">
```
Comment (jseppi) [inline data_explorer/templates/_footer.html]: eep, I should have caught this also, but I think we can simplify this markup a bit...I'll just make a PR to this branch

**Comment by tram** [comment]
Agree that we should be okay just having log out in the footer. I'll keep an eye on it as we launch and will add a question about it to post-launch testing.

**Pull request merged**